Minutes/2023/2023_08_07-minutes.md

```diff
@@ -0,0 +1,211 @@
+
+Ege Korkan: So for that we need to generate more content first to to show how it really looks like and something that can be certified because the single video or tutorial lighting at the moment would be too short.
+
+Justus Fries: ok
+
+Justus Fries: So and right now you have like what I saw on GitHub the the introduction right that does like the basic overview of what you can like do with it.
+
+Justus Fries: And so is there anything else that you have already finished or are you still working on? Like making it deeper?
+
+Idil Sezgin: actually published first two videos in the GitHub if I'm not mistaken.
+
+Idil Sezgin: First one is introduction to this series. Basically like what you need to expect from the series.
+
+Idil Sezgin: Like what is web of things.
+
+Idil Sezgin: we have 12 right now.
+
+Idil Sezgin: So basically, for the first six one we explained JSON and JSON schema.
+
+Idil Sezgin: Both in practice and the as a concept, and then we moved to the OSI layers and then we explain general generic web of things.
+
+Idil Sezgin: I'm still working on the new videos.
+
+Idil Sezgin: So for the website, I hope to have a first version of how it will look like by tpac.
+
+Ege Korkan: So we can actually add more things to preliminary stuff like you mentioned and think of a way how we can like work together.
+
+Ege Korkan: So you can provide the script or do the points and so on like what a voice over should generate.
+
+Justus Fries: How deep does it go. I guess the closest technology wise is kind of the JSON schema tutorial.
+
+> Ege Korkan: shows the slides
+
+Ege Korkan: And so like, it explains the concept like very basically and like the the simple types but I think we can also even go into like type generation like because we actually use it for an scripting API for type generation
+
+Ege Korkan: We need to talk about this combinations at the very end. There is the if then else on how we get a conditional validation.
+
+Justus Fries: Do you do that for all the parts that for all the topics?
+
+Ege Korkan: So the overall goal is to keep the videos short, which has multiple reasons.
+
+Ege Korkan: One is so that people can watch small things.
+
+Ege Korkan: The other thing it's it makes easy to store it in GitHub since you don't need to then find like a Google Drive.
+
+Justus Fries: ok right.
+
+Justus Fries: So the the task then is to, I guess, create the PowerPoint of the flow is that I create the PowerPoints.
+
+Justus Fries: I guess it's a script as well or.
+
+Ege Korkan: yes
+
+Ege Korkan: So currently the the script is written in the PowerPoint notes for us, but we can think of a different way, but in the end the script is.
+
+> Ege Korkan: shows voiceover tool
+
+Ege Korkan: So once we put a text in here and and then from every text again like it just the pronunciation or like how we can do the pauses and it will generate the voice so that you can like correctly overlay and time according to the video.
```

Comment:
```suggestion
Ege Korkan: So once we put a text in here, it is just setting the pronunciation or how to do the pauses and it will generate the voice so that you can correctly overlay and time according to the video.
```